### ISSUE TITLE
[4.11.3] GZDoom fatal error

### ISSUE THREAD

**Issue by TolgaKerem07** [pa1nki113r/Project_Brutality#1248]
It seems to happen randomly, I haven't figured out what causes it to crash yet

[CrashReport.zip](https://github.com/pa1nki113r/Project_Brutality/files/14959031/CrashReport.zip)

**Comment by FDinolfo** [comment]
I've actually been dealing with similar crashes with the same type of error for the past few days. Just a question, do you have UDB in the background?

**Comment by GrayFace** [comment]
For me it always crashes with `Could not create vulkan image: out of device memory` message.

**Comment by popguy12** [comment]
pretty sure thats just shit with the engine
ive had it happen several times over in VKDoom where it just stops loading sprites, and after a couple second, just straight crashes

dont think theres much we can do about it, GZ devs just have to get off their fuckin asses and fix asset loading issues and what not

**Comment by GrayFace** [comment]
Yes, had that happen in Ashes Hard Reset recently. Might be a regression in new GZDoom version.

**Comment by popguy12** [comment]
closing unless its continuing to happen, since we have moved on from 4.11
